showcase/src/styles/styles.scss

```diff
@@ -14,4 +14,5 @@ IMPORTANT: Stark styles are provided as SCSS styles so they should be imported i
 @import "../app/welcome/pages/news/news-page.component";
 @import "../app/welcome/components/news-item/news-item.component";
 @import "../app/demo-ui/pages/message-pane/demo-message-pane-page.component";
+@import "../app/demo-ui/pages/progress-indicator/demo-progress-indicator-page.component";
```

Comment:
See my previous comment about the styles. And, if there no need to convert it to a theme file, then I think we should put this import in the component directly (`styleUrls`)